### PR TITLE
Revert to polipo

### PR DESCRIPTION
**Pull request by ppp0**

### CODE REVIEW

**Comment by ppp0** [comment]
@kris-lab please review

File: spec/puppet/environments/default/manifests/site.pp [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   require 'puppet::common'
 
-  class { 'squid_deb_proxy':
+  class { 'polipo':
```
Comment (kris-lab) [inline spec/puppet/environments/default/manifests/site.pp]: There as also `idleTime => 0` before. Should be added as well?

Comment (ppp0) [inline spec/puppet/environments/default/manifests/site.pp]: Thanks!

**Comment by ppp0** [comment]
@kris-lab please review

**Comment by kris-lab** [comment]
lgtm

**Pull request merged**